packages/adapter-squid/src/types/ConfigCodecs.ts

```diff
@@ -2,14 +2,21 @@ import * as t from "io-ts";
 import { TypeOf } from "io-ts";
 import { StepType } from ".";
 
+/**
+ * Codec for the Squid adapter configuration.
+ */
+export const SquidAdapterConfigCodec = t.type({
+   integratorId: t.string,
+   baseUrl: t.string,
+});
+
 /**
  * Codec for the Bridge.Swap step configuration.
  */
 export const BridgeSwapConfigCodec = t.type({
    fromChain: t.number,
    fromToken: t.string,
-   fromAddress: t.string,
-   fromAmount: t.number,
+   fromAmount: t.string,
```

Comment:
just out of curiosity, why `string` here? seemed unintuitive at first glance

Reply:
For precision. We can't represent the amounts in the native number type without losing precision so it's safer to use string.